neuralop/losses/data_losses.py

```diff
@@ -313,101 +313,3 @@ def rel(self, x, y, vol_elm):
     
     def forward(self, y_pred, y, vol_elm, **kwargs):
         return self.rel(y_pred, y, vol_elm)
```

Comment:
Why was the rest deleted? 
`self.rel` doesn't seem defined. If we don't need this let's just delete the class alltogether